dmoj/cptbox/_cptbox.pyx

```diff
@@ -225,6 +225,22 @@ def memory_fd_seal(int fd):
 
 cdef class Process
 
+cdef class Debugger
+
+cdef class ArgAccessor:
+    cdef Debugger debugger
+    cdef object base
+
+    def __cinit__(self, Debugger debugger, object base):
+        self.debugger = debugger
+        self.base = base
+
+    def __getitem__(self, reg):
+        return getattr(self.debugger, '%s%d' % (self.base, reg))
```

Comment:
Why are you doing this in `_cptbox`? Maybe you should move this into `AdvancedDebugger` instead.